src/app/components/banner/banner.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Image from "next/image";
+import { useTranslation } from 'react-i18next';
+import Photo_banner from '../image/banner.png';
+import styles from './banner.module.scss';
+import { CatalogButton } from '../button';
+
+const Banner = () => {
+  const { t } = useTranslation('common');
+
+  return (
+    <div className={styles.banner}>
+      <div className={styles.banner__overlay}>
+        <div className={styles.banner__content}>
+          <h1>Бытовая химия, косметика и хозтовары</h1>
+          <p>Оптом по Кокшетау и области</p>
+          <CatalogButton className={styles.catalogButton} />
+        </div>
+
+        <div className={styles.banner__features}>
+          <div className={styles.feature}>
+            <div className={styles.plusCircle}>+</div>
+            <div className={styles.featureText}>
+              <p>Только самые</p>
+              <p>выгодные предложения</p>
+            </div>
+          </div>
+          <div className={styles.feature}>
+            <div className={styles.plusCircle}>+</div>
+            <div className={styles.featureText}>
+              <p>Бесплатная доставка</p>
+              <p>по Кокшетау от 10 тыс ₸</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className={styles.banner__image}>
+        <Image src={Photo_banner} alt="Cleaning Supplies" layout="fill" objectFit="cover" />
```

Comment:
```suggestion
        <Image 
            src={Photo_banner} 
            alt="Cleaning Supplies" 
            layout="fill" 
            objectFit="cover" 
        />
```
если у компонента или тега больше 1 атрибута, для читаемости переноси их на новую строку